dealii/petsc_trilinos/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+##
+#  CMake script for the petsc_trilinos tutorial program:
+##
+
+# Set the name of the project and target:
+set(TARGET "petsc_trilinos")
+
+# Declare all source files the target consists of. Here, this is only
+# the one step-X.cc file, but as you expand your project you may wish
+# to add other source files as well. If your project becomes much larger,
+# you may want to either replace the following statement by something like
+#  file(GLOB_RECURSE TARGET_SRC  "source/*.cc")
+#  file(GLOB_RECURSE TARGET_INC  "include/*.h")
+#  set(TARGET_SRC ${TARGET_SRC}  ${TARGET_INC})
+# or switch altogether to the large project CMakeLists.txt file discussed
+# in the "CMake in user projects" page accessible from the "User info"
+# page of the documentation.
+set(TARGET_SRC
+  ${TARGET}.cc
+  )
+
+# Usually, you will not need to modify anything beyond this point...
+
+cmake_minimum_required(VERSION 3.13.4)
+
+find_package(deal.II 9.5.0
+  HINTS ${deal.II_DIR} ${DEAL_II_DIR} ../ ../../ $ENV{DEAL_II_DIR}
+  )
```

Comment:
Can you please move the `find_package` command to the top-level CMake file similar to the other packages?

Also, if the version requirement of v9.5.0 fails (e.g. we use xsdk v0.8.0 and we have v9.4.0), we want the examples to be skipped instead of CMake failing (Deal.II can be enabled for other examples that work with older Deal.II version).

Thanks!

Reply:
merged changes to `testing-v0.4.0` branch

Reply:
These examples should actually also work with 9.4. Why not go with the lower required version number?

Reply:
Looks like you are commenting on outdated code. 🙂 I simply removed the version requirement.